README.md

```diff
@@ -59,9 +56,11 @@ npm install @brightspace-ui/core
 * Mixins
   * [ArrowKeysMixin](mixins/arrow-keys-mixin.md): manage focus with arrow keys
   * [AsyncContainerMixin](mixins/async-container/): manage collective async state
+  * [FormElementMixin](components/form/docs/form-element-mixin.md): allow components to participate in forms and validation
   * [LocalizeMixin](mixins/localize-mixin.md): localize text in your components
   * [ProviderMixin](mixins/provider-mixin.md): provide and consume data across elements in a DI-like fashion
   * [RtlMixin](mixins/rtl-mixin.md): enable components to define RTL styles
+  * [SkeletonMixin](components/skeleton/): make components skeleton-aware
```

Comment:
I added some special call-outs here for the mixin (and the skeleton one) despite these things living in `components`. I like the idea of grouping all the mixins together like this, but am fine to remove this if others disagree.